src/test/config/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+include_directories(${GLIB_INCLUDES})
+link_libraries(${GLIB_LIBRARIES})
+
+add_shadow_exe(test-config test_config.c)
+
+add_test(
+        NAME config-convert-xml2yaml
+        COMMAND /usr/bin/env python3 ${CMAKE_SOURCE_DIR}/src/tools/convert.py xml2yaml --output test.shadow.config.yaml ${CMAKE_CURRENT_SOURCE_DIR}/shadow.config.xml
+)
+
+add_test(
```

Comment:
Could use a comment briefly explaining that this is reversing the conversion of the previous test.